src/surabhi/assignment14/ShopUsingScanner.java

```diff
@@ -0,0 +1,152 @@
+package surabhi.assignment14;
+
+import java.util.Scanner;
+/*
+ * Assignment 14: Shop using user input (using scanner Class)
+ * Added new logic as if user input only 3 quantities not all then we can pass single parameter to see expected output and
+ *  avoided passing 0 in parameter
+ *  
+ * */
+
+public class ShopUsingScanner {
+	
+	int maggieQuantity=50, dosaQuantity=43, pouchesQuantity=39, panipuriQuantity=43, masalaQuantity=73;
+	
+	void requestedQMaggie(int maggieQ) {
+		if(maggieQ<=50 && maggieQ!=0 ) {
```

Comment:
this 50 should not be hardcoded, use variable. maggieQuantity where you have define 50